Makefile

```diff
@@ -10,6 +10,11 @@ XCARGS := -workspace $(WORKSPACE) \
 build:
 	set -o pipefail && xcodebuild $(XCARGS) -scheme VelocidiSDK build | xcpretty
 
+format:
+	set -o pipefail && \
```

Comment:
I don't think we need this here because we are not using pipes.

Reply:
Removed in 03ce0d1